client/plots/wsiviewer/test/wsiviewer.integration.spec.ts

```diff
@@ -0,0 +1,59 @@
+import * as helpers from '../../../test/front.helpers.js'
+import tape from 'tape'
+
+/**************
+Tests:
+	- Render wsiviewer as chartType
+
+***************/
+
+/*************************
+ reusable helper functions
+**************************/
+
+const runpp = helpers.getRunPp('mass', {
+	state: {
+		dslabel: 'nintendoPublic',
+		genome: 'mm10'
+	},
+	debug: 1
+})
+
+tape.skip('\n', function (test) {
+	test.pass('-***- plots/wsiviewer/wsiviewer -***-')
+	test.end()
+})
+
+//Data not available to run in Termdb test (i.e. CI)
```

Comment:
In order to make it work on CI, we would need to integrate a small svs or till to the repo, or download it during test execution. Also, we would need to start TileServer on CI, that's doable too. @creilly8 @xzhou82

Reply:
please do not enable it in CI for now

Reply:
Ok. It wouldn't be difficult to integrate it in the next steps.